openshift_scalability/utils.py

```diff
@@ -825,14 +824,15 @@ def pod_handler(inputpods, globalvars):
         pod_config["metadata"]["name"] = basename
 
         create_pods(pod_config, num,storagetype, globalvars)
-
-    if len(globalvars["pend_pods"]) > 0:
-        pod_data(globalvars)
-
-    if "podtuningset" in globalvars:
-        del(globalvars["podtuningset"])
-        del(globalvars["totalpods"])
-    del(globalvars["pend_pods"])
+    tolerate_bad_pods = globalvars["tolerate"]
```

Comment:
Local variables should be set at the beginning of the function, with the rest of them.